app/models/requests/request.rb

```diff
@@ -232,7 +232,7 @@ def load_items
       elsif visuals?
         return nil
       else
-        if @mfhd
+        if @mfhd && serial?
```

Comment:
The failing integration tests look like they are due to a syntax error in javascript. The other tests are likely failing because they were operating on the assumption they'd be examining only items attached to the mfhd value passed. Since we are now looking at all mfhds and item records for non serials some adjustments need to be made.